code/modules/atmospherics/machinery/components/fusion/hypertorus.dm

```diff
@@ -1,9 +1,9 @@
 ///Max amount of radiation that can be emitted per reaction cycle
 #define FUSION_RAD_MAX						5000
 ///Maximum instability before the reaction goes endothermic
-#define FUSION_INSTABILITY_ENDOTHERMALITY   7
+#define FUSION_INSTABILITY_ENDOTHERMALITY   5
 ///Maximum reachable fusion temperature
-#define FUSION_MAXIMUM_TEMPERATURE			1e30
+#define FUSION_MAXIMUM_TEMPERATURE			1e9
```

Comment:
Unless there is a good reason for it, limit this to 1e8 again please.

Reply:
putting at 1e8 will limit the reaction a lot, since it will remove one power level and it has other balancing implications. 1e9 is the minimum i could go without limiting it too much. Tho if you think that 1e9 is too high, i can try to lower it but i'll loose quite a bit of interactions

Reply:
Can you go into detail about what balancing implications and limits that would cause? Also, aren't you the one defining the constraints?

Reply:
mm there was the problem of the gas production and the damage calculations, but i suppose i can work around that... i'll try to set it to 1e8 then